action.yml

```diff
@@ -19,8 +19,11 @@ inputs:
     description: Should the setup.ini file signature be checked?
     required: false
     default: true
-  site:
-    description: Download site URL
+  pubkey:
+    description: URL or absolute path of extra public key file (RFC4880 format)
+    required: false
```

Comment:
It's bad that setup allows a URL here, and encouraging people to use it is potentially unsafe, so maybe this should just lie and say "absolute path"?